packages/lisk-p2p/src/validation.ts

```diff
@@ -76,21 +76,20 @@ export const validatePeerInfo = (
 			: 0;
 
 	const peerInfo: P2PDiscoveredPeerInfo = {
+		...protocolPeer,
 		ipAddress: protocolPeer.ip,
 		wsPort,
 		height,
 		os,
 		version,
-		options: {
-			httpPort: protocolPeer.httpPort,
-			broadhash: protocolPeer.broadhash,
-			nonce: protocolPeer.nonce,
-			...protocolPeer.options
-		},
-		isDiscoveredPeer: true,
+		httpPort: protocolPeer.httpPort,
+		broadhash: protocolPeer.broadhash,
+		nonce: protocolPeer.nonce,
```

Comment:
Do these fields still need to be listed explicitly? They should already be added as part of the expansion `...protocolPeer`.